charts/dapr/Chart.yaml

```diff
@@ -1,23 +1,23 @@
 apiVersion: v1
-appVersion: "1.2.1"
+appVersion: "1.3.0-rc.1"
 description: A Helm chart for Dapr on Kubernetes
 name: dapr
-version: 1.2.1
+version: 1.3.0-rc.1
 dependencies:
   - name: dapr_rbac
-    version: "1.2.1"
+    version: "1.3.0-rc.1"
     repository: "file://dapr_rbac"
   - name: dapr_operator
-    version: "1.2.1"
+    version: "1.3.0-rc.1"
     repository: "file://dapr_operator"
   - name: dapr_placement
-    version: "1.2.1"
+    version: "1.3.0-rc.1"
     repository: "file://dapr_placement"
   - name: dapr_sidecar_injector
-    version: "1.2.1"
+    version: "1.3.0-rc.1"
     repository: "file://dapr_sidecar_injector"
   - name: dapr_sentry
-    version: "1.2.1"
+    version: "1.3.0-rc.1"
     repository: "file://dapr_sentry"
   - name: dapr_dashboard
     version: "0.7.0"
```

Comment:
Is this a prior mistake?

Reply:
Yes

Reply:
Yes - and we may fix this by syncing dashboard and helm chart versions to 0.7.1 at final release time TBD